beacon-chain/db/block.go

```diff
@@ -180,6 +167,7 @@ func (db *BeaconDB) GetBlockBySlot(slot uint64) (*types.Block, error) {
 		if blockhash == nil {
 			return nil
 		}
+		log.Printf("block hash in db: %v", blockhash)
```

Comment:
Is this a debug statement? PLease set logging level to the appropriate level. (info/debug/etc)